draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -325,8 +332,12 @@ criteria in the remainder of this document might not apply.
 
 # Evaluation Criteria {#evaluation-criteria}
 
-As noted above, authors are expected to do a well-rounded evaluation of the pros
-and cons of congestion control algorithms that are brought to the IETF. The
+Authors are expected to evaluate the congestion control algorithms that
+they bring to the IETF.
```

Comment:
IMO this paragraph pins too much on the authors. "Authors are expected to evaluate..." and then the second sentence uses the passive voice. I would prefer a paragraph like so:

"Publication of a specification by the IETF requires a well-rounded evaluation of the pros and cons of an algorithm. Both the document authors and participants in an IETF Working Group can conduct parts of this evaluation."

That's all we need to say, but I wouldn't object to further advice saying that the more work the authors do, the more likely it is to progress.